ChangeLog.d/remove-via-padlock-support.txt

```diff
@@ -0,0 +1,5 @@
+Removals
+   * Drop support for VIA Padlock. Removes MBEDTLS_PADLOCK_C and
+   * associated code.
```

Comment:
This line and line 5 should probably not have the `*`. This might be the cause of the CI failure on the changelog entry (I haven't checked)

Reply:
Thank you!

Reply:
It wouldn't cause a CI failure. “associated code.” would be a separate changelog entry. We keep entries from the same file grouped so the two lines would even remain together.